app/src/main/java/br/com/compass/compassmart/ui/onboarding/OnboardingFragment.kt

```diff
@@ -0,0 +1,86 @@
+package br.com.compass.compassmart.ui.onboarding
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.widget.ViewPager2
+import br.com.compass.compassmart.R
+import br.com.compass.compassmart.databinding.FragmentOnboardingBinding
+import br.com.compass.compassmart.ui.onboarding.model.InformacoesModel
+
+class OnboardingFragment : Fragment() {
+
+    private val binding: FragmentOnboardingBinding by lazy {
+        FragmentOnboardingBinding.inflate(layoutInflater)
+    }
```

Comment:
Aqui eu passei o implementação errada do viewBinding. O matheus me sinalizou que esse lazy pode dar algum problema com os ciclos de vida da fragment.

```suggestion
    private var _binding: FragmentOnboardingBinding? = null
    private val binding get() = _binding!!
```

Reply:
Alterado.